src/mongo/editors/MongoCollectionNodeEditor.ts

```diff
@@ -11,8 +11,10 @@ import { getNodeEditorLabel } from '../../utils/vscodeUtils';
 
 export class MongoCollectionNodeEditor implements ICosmosEditor<IMongoDocument[]> {
     private _collectionNode: IAzureParentNode<MongoCollectionTreeItem>;
+    private EJSON;
     constructor(collectionNode: IAzureParentNode<MongoCollectionTreeItem>) {
         this._collectionNode = collectionNode;
+        this.EJSON = require('mongodb-extended-json');
```

Comment:
Why aren't you using import?

Reply:
mongodb-extended-JSON doesn't have typings (no index.d.ts, or an entry in the [@types](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types) repo) 

This was the `non-ideal` code I was talking about yesterday

Reply:
doesn't mean you can't import at the top:

import * as EJSON = require("...");

Reply:
The above seems to throw a ts error. 
`import EJSON = require('mongodb-extended-json')` throws a typings error. 

After discussing with @EricJizbaMSFT , I'm going with `const EJSON = require('mongodb-extended-json')`